groups/promo-infrastructure/locals.tf

```diff
@@ -6,7 +6,29 @@ locals {
   s3_origin_id                     = "promo"
   s3_promo_cf_logs_bucket          = "${var.aws_account}.${var.aws_region}.promo-s3-cloudfront-logs"
   s3_promo_web_hosting_logs_bucket = "${var.aws_account}.${var.aws_region}.promo-s3-web-hosting-logs"
-  cw_promo_log_grp                 = "promo-cw-lg"
+  account_ids                      = data.vault_generic_secret.account_ids.data
+  security_s3                      = data.vault_generic_secret.security_s3.data
+
+  kms_customer_master_keys = {
+    ssm = {
+      description             = "Session Manager data encryption key"
+      deletion_window_in_days = 30
+      enable_key_rotation     = true
+      is_enabled              = true
+      key_usage_foreign_account_ids = [
+        local.account_ids[var.aws_account]
```

Comment:
Unneeded? This is adding the local account into the key policy, but that already exists by default. This is only required to add cross account permissions for KMS keys.

Reply:
This has been removed